engines/dotnet/Main.cs

```diff
@@ -1,10 +1,9 @@
-﻿using System;
-using System.IO;
+﻿using System.Diagnostics;
+using System.Text;
 using System.Text.RegularExpressions;
-using System.Diagnostics;
 
-// The configuration of a benchmark, as parsed from the incoming KLV data.
-public struct Config
+/// <summary>The configuration of a benchmark, as parsed from the incoming KLV data.</summary>
+struct Config
```

Comment:
I'd say this probably should be a `class` instead, given its size.

Also, the code would be more idiomatic if the fields were turned into PascalCased properties.